docu/files/SkeletorInBuilderCLI.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Design a Builder UI with Skeletor'
+service: 'Builder SDK'
+area: 'Core'
+type: Tutorial
+order: -
+---
+
+The Builder CLI features the Skeletor tool with which you can easily create views just by dragging and dropping UI components. <br><br>
+
+<h3> How to use the Skeletor</h3>
+
+
+1. Open a terminal window.
+
+ <img src="img/builder_cli_menu_runSkeletor.png"><br><br>
+
+2. Run the command ```builder runSkeletor``` to start the Skeletor. The HTTP server will start locally.
+3. Enter the URL http://localhost:8082 in a browser.  The Skeletor design interface will appear. 
+4. To design a view, drag and drop the components from the left of the screen onto the main screen. Organize the components as they should appear in the Builder UI. 
+
+  <img src="img/skeletor_ui_drag.png" style="width:600px" class="img-click-modal" alt="The Skeletor UI"/><br><br>
+5. After you have created the view, you can preview your results by clicking the preview button to the right of the delete icon.  The view it will be immediately stored in the local cache of your browser. That means you’ll never loose the view until you delete it by explicitly pressing the delete button.
```

Comment:
Rewrite passive voice sentences, for example, "have created," throughout the document to use present tense, active voice. For example: "After you create the view, click **Preview** to review your results." 

Also, I'm not looking at the image, so I'm not sure what it looks like, but just as a reminder: You can use "the button" only if you refer to an unlabeled icon. If the button has a label, format the button/icon name in bold as shown in the preceding example.